conwaylib/Game.cs

```diff
@@ -26,13 +28,13 @@ public Game(IReadableBoard initialBoard, IEvolution evolution)
     /// </summary>
     public IReadableBoard Turn()
     {
-      for (int x = 0; x < mCurBoard.Width; ++x)
+      Parallel.For(0, mCurBoard.Width, (x, state)=>
```

Comment:
The `state` variable in the lambda is not needed - there is an overload that does not include the loop state